Configuration/AlCa/python/autoCond.py

```diff
@@ -20,9 +20,11 @@
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
     'run2_mc_hi'        :   '80X_mcRun2_HeavyIon_v3',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '80X_dataRun2_v4',
+    'run1_data'         :   '80X_dataRun2_v5',
```

Comment:
it's more appropriate to remove / revert the commit that edits this file.
If I understand correctly, the needed GT version is already in the release and the github web display just gets confused with the changes actually made.